docs/docs/migrate/keptnapp/index.md

```diff
@@ -0,0 +1,60 @@
+# Migrating KeptnApp to KeptnAppContext
+
+Keptn has recently orphaned the concept of manually created KeptnApp
+and instead uses the KeptnApp auto-discovery feature combined with
+the newly introduced KeptnAppContext custom resource.
+
+## Migration steps
+
+> **Warning**
+Please note that the migration is needed to be executed only for
+**manually created KeptnApp** resources.
+KeptnApp resources created by
+the automatic app-discovery feature do not need any actions.
```

Comment:
```suggestion

> **Note**
Migration is only required for
**manually created KeptnApp** resources.
KeptnApp resources created by
the
[automatic app-discovery](../guides/auto-app-discovery)
feature do not require modification
unless you edited them manually to add pre/post tasks and evaluations.

## Migration steps
```
* I don't think this warrants a "Warning" -- that should be reserved for actions that have dire consequences
* The admonition should come before the section that gives how-to info
* I added the "unless" but I don't know if that is true.  But we need to address the hybrid approach that we've been recommending, to auto-generate KeptnApp then edit it to add tasks and evaluations

Reply:
I would stick to warning here, as it's very important that the users know that they need to migrate only if the KeptnApp was created manually. Just for visibility